### PR TITLE
Fix BuildKit buikld for GitRepo without .git suffix

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

**Comment by tianon** [comment]
I'm still of the opinion that this is a bug in either our tooling or BuildKit + Docker CLI/Engine, since `git clone` works just fine without the `.git` suffix. :sweat_smile:

**Comment by tianon** [comment]
(in other words, I think https://github.com/docker-library/meta-scripts/pull/15 is a very sane short and medium -term solution)

**Comment by LaurentGoderre** [comment]
It was still valuable as a backup

File: library/ubuntu [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # see also https://wiki.ubuntu.com/Releases#Current
 
 Maintainers: Tomáš Virtus <tomas.virtus@canonical.com> (@woky), Cristóvão Cordeiro <cristovao.cordeiro@canonical.com> (@cjdcordeiro)
-GitRepo: https://git.launchpad.net/cloud-images/+oci/ubuntu-base
+GitRepo: https://git.launchpad.net/cloud-images/+oci/ubuntu-base.git
```
Comment (tianon) [inline library/ubuntu:5]: This one is actually the example URL I gave to the Moby folks internally -- Launchpad canonically does not use `.git` and does not support it (hence the CI failures on this PR :smile:)

**Comment by tianon** [comment]
https://github.com/docker-library/meta-scripts/pull/15 was successful :tada: :heart:

**Pull request closed without merging**